doc/api/diagnostics_channel.md

```diff
@@ -409,6 +459,534 @@ channel.subscribe(onMessage);
+This is the same as calling [`channel.subscribe(onMessage)`][] on each channel
+individually.
+
+```mjs
+import diagnostics_channel from 'node:diagnostics_channel';
+
+const channels = diagnostics_channel.tracingChannel('my-channel');
+
+channels.subscribe({
+  start(message) {
+    // Handle start message
+  },
+  end(message) {
+    // Handle end message
+  },
+  asyncStart(message) {
+    // Handle asyncEnd message
+  },
+  asyncEnd(message) {
+    // Handle asyncEnd message
+  },
+  error(message) {
+    // Handle error message
+  },
+});
+```
+
+```cjs
+const diagnostics_channel = require('node:diagnostics_channel');
+
+const channels = diagnostics_channel.tracingChannel('my-channel');
+
+channels.subscribe({
+  start(message) {
+    // Handle start message
+  },
+  end(message) {
+    // Handle end message
+  },
+  asyncStart(message) {
+    // Handle asyncEnd message
+  },
+  asyncEnd(message) {
+    // Handle asyncEnd message
+  },
+  error(message) {
+    // Handle error message
+  },
+});
+```
+
+#### `tracingChannel.unsubscribe(subscribers)`
+
+<!-- YAML
+added:
+ - REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `subscribers` {Object} Set of [TracingChannel Channels][] subscribers
+  * `start` {Function} The [`start` event][] subscriber
+  * `end` {Function} The [`end` event][] subscriber
+  * `asyncStart` {Function} The [`asyncStart` event][] subscriber
+  * `asyncEnd` {Function} The [`asyncEnd` event][] subscriber
+  * `error` {Function} The [`error` event][] subscriber
+* Returns: {boolean} `true` if all handlers were successfully unsubscribed,
+  and `false` otherwise.
+
+Helper to unsubscribe a collection of functions from the corresponding channels.
+This is the same as calling [`channel.unsubscribe(onMessage)`][] on each channel
+individually.
+
+```mjs
+import diagnostics_channel from 'node:diagnostics_channel';
+
+const channels = diagnostics_channel.tracingChannel('my-channel');
+
+channels.unsubscribe({
+  start(message) {
+    // Handle start message
+  },
+  end(message) {
+    // Handle end message
+  },
+  asyncStart(message) {
+    // Handle asyncEnd message
+  },
+  asyncEnd(message) {
+    // Handle asyncEnd message
+  },
+  error(message) {
+    // Handle error message
+  },
+});
+```
+
+```cjs
+const diagnostics_channel = require('node:diagnostics_channel');
+
+const channels = diagnostics_channel.tracingChannel('my-channel');
+
+channels.unsubscribe({
+  start(message) {
+    // Handle start message
+  },
+  end(message) {
+    // Handle end message
+  },
+  asyncStart(message) {
+    // Handle asyncEnd message
+  },
+  asyncEnd(message) {
+    // Handle asyncEnd message
+  },
+  error(message) {
+    // Handle error message
+  },
+});
+```
+
+#### `tracingChannel.traceSync(fn[, data[, thisArg[, ...args]]])`
+
+<!-- YAML
+added:
+ - REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `fn` {Function} Function to wrap a trace around
+* `data` {Object} Shared object to correlate trace events through
```

Comment:
A section explaining the utility of shared data objects would be useful.

Also, `import 'node:trace_events'` is a thing, so there's a terminology collision here.